rmw_connext_cpp/src/rmw_event.cpp

```diff
@@ -20,6 +20,55 @@
 
 extern "C"
 {
+
+/// Initialize a rmw publisher event.
+/**
+ * \param[in|out] rmw_event to initialize
+ * \param publisher to initialize with
+ * \param event_type for the event to handle
+ * \return `RMW_RET_OK` if successful, or
+ * \return `RMW_RET_INVALID_ARGUMENT` if invalid argument, or
+ * \return `RMW_RET_UNSUPPORTED` if event_type is not supported, or
+ * \return `RMW_RET_ERROR` if an unexpected error occurs.
+ */
```

Comment:
The docstrings here are redundant are they not?

Reply:
I agree. I saw it was done for `rmw_take_event()`, so I did it for `rmw_publisher_event_init()` and `rmw_subscription_event_init()` as well. I'll remove all of them.